docs/src/pages/regions-and-acl.mdx

```diff
@@ -0,0 +1,76 @@
+import { Callout } from "nextra-theme-docs";
+
+## Regions
+
+UploadThing allows you to select where your files are stored by configuring your
+app's region on the UploadThing dashboard. If no configuration is done, your
+files are uploaded to our default region which is currently `AWS us-west-2`.
+
+<Callout type="info">
+  Uploading to a region different than the default one requires a paid plan.
+</Callout>
+
+### Configuring region
+
+Go to your app's settings on the UploadThing dashboard. Under `Regions and ACL`,
+you can select the region you want to upload your files to. Once changed, all
+new file uploads will go to the selected region.
+
+import Image from "next/image";
+
+<Image
+  src="/region-settings.png"
+  alt="Region settings"
+  width="1546"
+  height="998"
+  style={{ marginTop: "1rem", borderRadius: "0.5rem" }}
+/>
+
+<Callout type="info">
+  Changing your app's region will not move your existing files to the new
+  region, it will only affect new files.
```

Comment:
```suggestion
  region, it will only affect new files. For assistance migrating uploaded files to a new region, 
  please reach out in discord, or via email (ut-support@ping.gg)
```